nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -81,6 +81,12 @@
 
     protected String name;
 
+    /**
+     * @since 2.3
+     */
+    protected String isProxy;
+
+
     public Document(String id, String type, List<String> facets, String changeToken, String path, String state,
```

Comment:
This constructor should delegate to the new one, passing `null` for `isProxy`.
It should also probably be deprecated as it's not used anymore in our code, and the `isProxy` field should be there for this client version as it's only compatible with LTS 2016+.